### PR TITLE
Bug 1333101 Use long date format on logs to remove ambiguity in month vs day

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1333101

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: app/views/browse/build.html [hunk 1]
```diff
@@ -123,9 +123,9 @@
                         <span class="space-after">{{build.status.phase}}</span>
 
                         <div ng-if="build.status.startTimestamp && !logEmpty" class="log-timestamps">
-                          Log from {{build.status.startTimestamp  | date : 'short'}}
+                          Log from {{build.status.startTimestamp  | date : 'MMM dd, yyyy h:mm a'}}
```
Comment (spadgett) [inline app/views/browse/build.html]: Should we use `date : 'medium'` which is the almost the same, but will be localized?

https://docs.angularjs.org/api/ng/filter/date

Comment (jwforres) [inline app/views/browse/build.html]: agreed, updating the other places we were using this

**Comment by jwforres** [comment]
@spadgett updated everything to use medium

**Comment by jwforres** [comment]
[merge]

**Pull request merged**